README.md

```diff
@@ -159,7 +159,7 @@ end
 
 Grocery.new()
 #=> %Grocery{
-#     item: {"Apple", 4, false}
+#     item: {"Orange", 4, false}
```

Comment:
This won't actually produce this output. You need to update the definition's overwrite as well.

Reply:
Ok, I will overwrite that as well! Thank you for the patience 🙇